_config.yml

```diff
@@ -1,8 +1,8 @@
 # Site settings
-title:  U.S. Web Design Standards
-description: The Standards provide research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
+title:  U.S. Web Design System
+description: The Design System provides research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
```

Comment:
I wonder if this can be reworked again, to something like:

**Design and build fast, accessible, mobile-friendly government websites backed by user research.**

Reply:
Updated in 060cb04.